schema/workbook.json

```diff
@@ -375,11 +354,88 @@
                             "type": "string",
                             "description": "The visualization type of the step",
                             "examples": ["table", "tiles"]
+                        },
+                        "mapSettings": {
+                            "$ref":"#/definitions/mapSettings"
                         }
                     }
                 },
-                "title": {
-                    "$ref": "#/definitions/title"
+                "name": {
+                    "$ref": "#/definitions/name"
+                },
+                "customWidth": {
+                    "$ref": "#/definitions/customWidth"
+                },
+                "showPin": {
+                    "$ref": "#/definitions/showPin"
+                },
+                "styleSettings": {
+                    "$ref": "#/definitions/styleSettings"
+                },
+                "conditionalVisibility": {
+                    "$ref": "#/definitions/conditionalVisibility"
+                }
+            }
+        },
+        "group": {
```

Comment:
some of this schema changes are from workbook groups, so that PR probably should get merged in first?

Reply:
that pr is done